### ISSUE TITLE
[BITV/A11Y] Multiselect accessibility

### ISSUE THREAD

**Issue by PVince81** [nextcloud/nextcloud-vue#2504]
From workshop:
  * [x] See <https://www.w3.org/TR/wai-aria-practices-1.1/examples/combobox/aria1.1pattern/listbox-combo.html>
  * [x] Focus needs to be clearly visible
  * [x] Icons on right need to be read out, and should be grouped with the name

Found by Vincent in another ticket:
- [x] multiselect should enable focus mode like other controls ? (still in browse mode when entering with tab)
- [x] multiselect entries are not read when moving with arrow keys in share dropdown (observed in right sidebar in files app)
- [x] multiselect should set the aria-describedby attribute to the input instead of leaving it on the main div? (aria-labelledby already present in vue-select)

More issues:
- [x] https://github.com/nextcloud/nextcloud-vue/issues/2951

**Comment by PVince81** [comment]
any taker ? (even if just sub-items) @skjnldsv @Pytal @vinicius73 @raimund-schluessler

**Comment by PVince81** [comment]
as discussed, it is likely that we'll need to find a new library

@CarlSchwan pointed out that vue-select might be suitable, however we will likely need to contribute accessibility fixes upstream

**Comment by Pytal** [comment]
After searching around it seems the aforementioned https://github.com/sagalbot/vue-select is the best library for our purposes, the API seems pretty extensible and covers all if not most of our existing use cases as far as I can see

**Comment by PVince81** [comment]
note: will need backport to 25 (the matching nextcloud-vue version used there)

**Comment by Pytal** [comment]
WIP https://github.com/nextcloud/nextcloud-vue/pull/3435

**Comment by Pytal** [comment]
Transferred "After adding a person/group, focus should stay on the input field"
- https://github.com/nextcloud/nextcloud-vue/issues/3570

**Comment by Pytal** [comment]
Transferred "User status dialog "Clear status after" menu entries need border for focus feedback"
- https://github.com/nextcloud/server/issues/35763
